spec.html

```diff
@@ -8940,11 +8941,11 @@ <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
 
       <emu-clause id="sec-arraycreate" aoid="ArrayCreate">
         <h1>ArrayCreate ( _length_ [ , _proto_ ] )</h1>
-        <p>The abstract operation ArrayCreate takes argument _length_ (a non-negative integer) and optional argument _proto_. It is used to specify the creation of new Array exotic objects. It performs the following steps when called:</p>
+        <p>The abstract operation ArrayCreate takes argument _length_ (a non-negative integral Number) and optional argument _proto_. It is used to specify the creation of new Array exotic objects. It performs the following steps when called:</p>
```

Comment:
What is a "non-negative integral Number"? With real numbers it's clear: 0 is non-negative. But is the Number -0 negative or not?

Reply:
We need to define it, as we are doing for `integral Number`. Would it be a problem if we define that `non-negative integral Number` is integral Numbers greater than *+0*, considering that *+0* is greater than *-0?*

Reply:
@caiolima +0 is not greater than -0. I would imagine the term "non-negative" when referring to integral Numbers simply refers to whether the sign bit is set. So this would exclude -0. If we want to include -0, I would just add "or -0". But we could use mathematical values here instead, right?